f5/bigip/resource.py

```diff
@@ -161,6 +161,11 @@ class UnsupportedOperation(F5SDKError):
     pass
 
 
+class InvalidStatsJsonReturned(KeyError):
+    """Returned stats JSON should always contain 'entries' key"""
+    pass
+
```

Comment:
We can handle those kind of exceptions when we start enabling this on each submodule, for now I can see LTM stats follow the pattern I set above.